### PR TITLE
Fix some more tests, disable 27

### PR DESCRIPTION
**Pull request by rchiodo**

### CODE REVIEW

File: Python/Product/PythonTools/PythonTools/Commands/ExecuteInReplCommand.cs [hunk 1]
```diff
@@ -211,7 +211,7 @@ internal sealed class ExecuteInReplCommand : Command {
             var eval = (IPythonInteractiveEvaluator)window.InteractiveWindow.Evaluator;
 
             // The interpreter may take some time to startup, do this off the UI thread.
-            await ThreadHelper.JoinableTaskFactory.RunAsync(async () => {
+            await _serviceProvider.GetPythonToolsService().UIThread.InvokeAsync(async () => {
```
Comment (bschnurr) [inline Python/Product/PythonTools/PythonTools/Commands/ExecuteInReplCommand.cs:214]: what's the benefit here?

File: Python/Product/TestAdapter/testlauncher.py [hunk 2]
```diff
@@ -177,7 +177,8 @@ def collectwapper():
             notfound = getattr(collector, '_notfound', [])
             if notfound:
                   for arg, exc in notfound: 
-                      line = "(no name {!r} in any of {!r})".format(arg, exc.args[0])
+                      arg_zero = exc[0] if isinstance(exc, list) else exc.args[0]
+                      line = "(no name {!r} in any of {!r})".format(arg, arg_zero)
```
Comment (bschnurr) [inline Python/Product/TestAdapter/testlauncher.py:181]: "exc[0] if isinstance(exc, list) else exc.args[0]"  wow this looks like python

File: Python/Product/PythonTools/PythonTools/Commands/ExecuteInReplCommand.cs (hunk 1 above)
Comment (rchiodo) [reply Python/Product/PythonTools/PythonTools/Commands/ExecuteInReplCommand.cs:214]: ThreadHelper library isn't available during unit tests. It was causing exceptions when running.

Comment (rchiodo) [reply Python/Product/PythonTools/PythonTools/Commands/ExecuteInReplCommand.cs:214]: PythonToolsService.UIThread is using a MockUIThread during testing.

File: Python/Product/TestAdapter/testlauncher.py (hunk 2 above)
Comment (rchiodo) [reply Python/Product/TestAdapter/testlauncher.py:181]: It's a python file? Were you commenting on how different python looks than C#?

**Review by bschnurr — APPROVED** [review]

**Pull request merged**